jupyter/Dockerfile

```diff
@@ -9,7 +9,8 @@ RUN pip install jupyter_contrib_nbextensions \
   && jupyter contrib nbextension install --user
 
 COPY requirements.txt /tmp/requirements.txt
-
+RUN pip uninstall numpy -y
+RUN pip uninstall numpy -y
```

Comment:
Shouldn't it be enough to remove numpy once?

Reply:
unfortunately, no :D